### PR TITLE
Migrated TC test_remove_brick_while_rebalance_is_running.py

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
Migrated TC[ test_remove_brick_while_rebalance_is_running.py](https://github.com/gluster/glusto-tests/blob/master/tests/functional/dht/test_remove_brick_while_rebalance_is_running.py)

Updates: #292
Signed-off-by: Sheetal Pamecha <spamecha@redhat.com>

### CODE REVIEW

**Comment by Sheetalpamecha** [comment]
```
Results for dht
Test nature : disruptive
TestName : test_remove_brick_while_rebalance_is_running
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:02:43       |      NA     |
|     rep     |     PASS    |        00:03:15       |      NA     |
|     disp    |     PASS    |        00:01:58       |      NA     |
|   dist-rep  |     PASS    |        00:06:45       |      NA     |
|  dist-disp  |     PASS    |        00:03:06       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   5   |
|            Tests Skipped             |   0   |
|             Tests Passed             |   5   |
|             Tests Failed             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 18 minutes 01 seconds
✔ Tearing down successful.
```

File: tests/functional/dht/test_remove_brick_while_rebalance_is_running.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        try:
+            status_info = (self.
+                           redant.get_rebalance_status(self.vol_name,
+                                                       self.server_list[0]))
+            status = status_info['aggregate']['statusStr']
+            if 'in progress' in status:
+                # Stop rebalance on the volume
+                ret = self.redant.rebalance_stop(self.vol_name,
+                                                 self.server_list[0])
+                if ret['msg']['opRet'] != 0:
```
Comment (nik-redhat) [inline tests/functional/dht/test_remove_brick_while_rebalance_is_running.py:42]: It should be `'0'` we get a string value for opRet.

File: tests/functional/dht/test_remove_brick_while_rebalance_is_running.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                                      self.server_list[0], self.vol_name,
+                                      subvol_num=1))
+            if self.remove_brick_list is None:
+                raise Exception("Failed to form bricks list for shrink")
+
+            # Shrinking volume by removing bricks
+            ret = redant.remove_brick(self.server_list[0], self.vol_name,
+                                      self.remove_brick_list, "start",
+                                      excep=False)
+            if ret['msg']['opRet'] == 0:
```
Comment (nik-redhat) [inline tests/functional/dht/test_remove_brick_while_rebalance_is_running.py:125]: Same as above

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/dht/test_remove_brick_while_rebalance_is_running.py (hunk 2 above)
Comment (Sheetalpamecha) [reply tests/functional/dht/test_remove_brick_while_rebalance_is_running.py]: Done

**Comment by nik-redhat** [comment]
Could you please add the results again, considering the type of check would have still passed in case the value was == 0?

**Comment by Sheetalpamecha** [comment]
> Could you please add the results again, considering the type of check would have still passed in case the value was == 0?

Yes I intend to. Once shared machine is available I will run TCs

**Comment by Sheetalpamecha** [comment]
```
Results for dht
Test nature : disruptive
TestName : test_remove_brick_while_rebalance_is_running
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:03:15       |      NA     |
|     rep     |     PASS    |        00:02:38       |      NA     |
|     disp    |     PASS    |        00:02:02       |      NA     |
|   dist-rep  |     PASS    |        00:07:16       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   4   |
|            Tests Skipped             |   0   |
|             Tests Passed             |   4   |
|             Tests Failed             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 18 minutes 33 seconds
✔ Tearing down successful.

```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**